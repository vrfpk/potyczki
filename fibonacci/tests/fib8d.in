3552113168721905
