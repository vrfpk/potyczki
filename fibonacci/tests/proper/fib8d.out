15000000000003095
